src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable no-unused-expressions */
+/* eslint-disable react/jsx-one-expression-per-line */
+export function Person({ person }) {
+  let marriage = 'I am not married';
+
+  if (person.isMarried) {
+    person.sex === 'm'
+      ? marriage = `${person.partnerName} is my wife`
+      : marriage = `${person.partnerName} is my husband`;
+  }
+
+  if (person.isMarried && person.sex === 'm') {
+    marriage = `${person.partnerName} is my wife`;
+  }
+
+  if (person.isMarried && person.sex === 'f') {
+    marriage = `${person.partnerName} is my husband`;
+  }
```

Comment:
I don't understand this.
Why u do the same operation twice? Can you elaborate?

Looks like in 6-10 you assign `${person.partnerName} is my husband / wife`; 

and then you do the same

Reply:
I forgot to remove the old code